src/api/query/accountInfo.ts

```diff
@@ -0,0 +1,41 @@
+import { state } from '@/store/state';
+import { getters } from '@/store/getters';
+import { AccountInfo, Address, Balance } from '@/api/models';
+import { onUnmounted, Ref } from 'vue';
+
+export async function subscribeAccountInfo(account: Address, ref: Ref<AccountInfo>) {
+    const api = getters.api(state);
+    const unsub = await api.query.system.account(account, (res) => {
+        // TODO: Validation
+        const accountInfo = (res.toJSON() as unknown) as AccountInfo;
+        ref.value = accountInfo;
+    });
+    return unsub;
+}
+
+export async function getAccountInfo(account: Address) {
+    const api = getters.api(state);
+    const accountInfo = ((
+        await api.query.system.account(account)
+    ).toJSON() as unknown) as AccountInfo;
+    return accountInfo;
+}
+
+export async function getBalance(account: Address) {
+    const api = getters.api(state);
+    const accountInfo = await api.query.system.account(account);
+    return accountInfo.data.free.toBn();
+}
+
+export function subscribeBalance(account: Address, balance: Ref<Balance>) {
+    const api = getters.api(state);
+    const unsub = api.query.system.account(account, (res) => {
+        // TODO: Validation
+        balance.value = res.data.free.toBn();
+    });
+    onUnmounted(() => {
```

Comment:
this function will never be called because it has no context or `target` of any component (https://github.com/vuejs/vue-next/blob/052a621762c5f7c420464747ebbbed27c7350593/packages/runtime-core/src/apiLifecycle.ts#L67). In other words, this must explicitly pass the context of the current component or it must be called within the `setup()` function for it to create a hook